src/components/ReportActionItem/MoneyReportView.tsx

```diff
@@ -118,7 +121,7 @@ function MoneyReportView({report, policy, isCombinedReport = false, shouldShowTo
 
                                 return (
                                     <OfflineWithFeedback
-                                        pendingAction={report.pendingFields?.[fieldKey]}
+                                        pendingAction={pendingAction ? undefined : report.pendingFields?.[fieldKey]}
```

Comment:
Can we add a comment to explain why we return `undefined` if there is no pendingAction?

Reply:
We need to return the pending field action here as `undefined` if the `pendingAction` is not undefined to avoid the duplicate pending action because if `pendingAction` exists, we already wrapped `OfflineWithFeedback` outside the `MoneyRequestView`

Reply:
Gotcha, can we add this context as a comment in the code? I don't think it would be obvious to the next person who reads this code.